createdisk.sh

```diff
@@ -296,21 +327,25 @@ ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo systemctl enable io.podman
 ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo crictl stopp $(sudo crictl pods -q)'
 ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'for i in {1..3}; do sudo crictl rmp $(sudo crictl pods -q) && break || sleep 2; done'
 
-# Remove pull secret from the VM
-${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo rm -f /var/lib/kubelet/config.json'
 
-# Download the hyperV daemons and libvarlink-util dependency on host
-mkdir $1/packages
-sudo yum install -y --downloadonly --downloaddir $1/packages hyperv-daemons libvarlink-util
+if [ ${OKD_INSTALL} == false ]; then
```

Comment:
This can be replaced with `if ! ${OKD_INSTALL}`

Reply:
Thanks, so all the `if [ ${OKD_INSTALL} == false ]` can be replaced with this.. _shellNoob_ 🙅

Reply:
so `if [ ! {OKD_INSTALL} ]` evaluates to true if a value is set for `OKD_INSTALL` doesn't matter `false` or `true` i don't want that, so we have to explicitly check with `if [  {OKD_INSTALL} == false ]`

Reply:
I think I would also favour doing tests for variable empty/not empty rather than checking explicitly for `true`/`false`